llama_index/llms/base.py

```diff
@@ -76,17 +76,39 @@ def __str__(self) -> str:
 
 
 class LLMMetadata(BaseModel):
-    """LLM metadata."""
-
-    context_window: int = DEFAULT_CONTEXT_WINDOW
-    num_output: int = DEFAULT_NUM_OUTPUTS
-    is_chat_model: bool = False
-    is_function_calling_model: bool = False
-    # By default we don't know the model name. We can set it automatically for
-    # some types, but custom predictors (like locally loaded models) we won't
-    # know.
-    # Used for tests, logging, and sanity checks
-    model_name: str = "unknown"
+    context_window: int = Field(
+        default=DEFAULT_CONTEXT_WINDOW,
+        description="Number of tokens the model can intake when generating a response.",
```

Comment:
nit: context_window actually limits the total input + output tokens, not just the input.

Reply:
Thanks for the nit, implemented!